build.sbt

```diff
@@ -41,7 +41,6 @@ lazy val root = (project in file("."))
 lazy val app = (project in file("app"))
   .disablePlugins(BintrayPlugin)
   .enablePlugins(ConscriptPlugin, BuildInfoPlugin, SonatypePublish)
-//  .enablePlugins(BuildInfoPlugin, SonatypePublish)
```

Comment:
Why disable these plugins?

Reply:
The build breaks in Intellij whenever I load the project with the Conscript plugin. Would raise an issue but I'm not sure whether it's an issue in conscript or intellij. Haven't had much of a chance to look into it.

Reply:
Note, this is just removing a commented out line that I'd added during development. It's not actually a change to the build